src/ReactiveDomain.Policy.Tests/ReactiveDomain.Policy.Tests.csproj

```diff
@@ -9,14 +9,14 @@
     </ItemGroup>
   
     <ItemGroup>
-      <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-      <PackageReference Include="xunit" Version="2.4.1" />
-      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
-      <PackageReference Include="xunit.runner.console" Version="2.4.1">
+      <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
+      <PackageReference Include="xunit" Version="2.6.1" />
+      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
+      <PackageReference Include="xunit.runner.console" Version="2.4.2">
         <PrivateAssets>all</PrivateAssets>
         <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       </PackageReference>
-      <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" Condition="'$(TargetFramework)'!='netstandard2.1'">
+      <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5">
```

Comment:
We've updated the core xunit library to the latest; can we update the xunit runners to the latest versions as well?